src/hooks/sourceNodes/index.js

```diff
@@ -72,6 +72,51 @@ module.exports = async (
             { deserializeResponse: false, allPages: true },
           );
           if (payload) {
+            // `rich_text`, `links`, `link` fields link to other entities and we need to
+            // fetch them separately to make sure we have all the data
+            const linkedEntitiesIdsToFetch = payload.data.reduce(
+              (acc, payload) => {
+                const item_type_rel = payload.relationships.item_type.data;
+                const itemTypeForThis = loader.entitiesRepo.findEntity(
+                  item_type_rel.type,
+                  item_type_rel.id,
+                );
+                const fieldsToResolve = itemTypeForThis.fields.filter(
+                  fieldDef =>
+                    [`rich_text`, `links`, `link`].includes(fieldDef.fieldType),
+                );
+
+                const fieldsToResolveKeys = fieldsToResolve.map(
+                  field => field.apiKey,
+                );
```

Comment:
This part looks to get list of fields that response will have only ids and we should pull full entity information